tests/common.js

```diff
@@ -1,24 +1,33 @@
-var inkPath = '../engine/runtime.js'
+let path = require('path');
+
+var rootDir = path.join(__dirname, '..');
+var inkPath = path.join(rootDir, 'engine', 'runtime.js');
```

Comment:
Nit, but we'd better use `let` everywhere while we're at it.

Reply:
We can use `let` everywhere. I hardly ever write `var` anyway, I don't know why it's there to be honest 😅. I'll update the PR!